### PR TITLE
Support cupyx namespace

### PR DESCRIPTION
**Pull request by kmaehashi**
Depends on https://github.com/cupy/cupy/pull/894

### CODE REVIEW

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by okuta** [comment]
flake8
```
./chainer/backends/cuda.py:42:5: F401 'cupyx' imported but unused
./chainer/backends/cuda.py:43:5: H306  imports not in alphabetical order (cupyx, cupy.cuda)
```

**Comment by kmaehashi** [comment]
Fixed.

File: chainer/functions/array/resize_images.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import cupyx
```
Comment (okuta) [inline chainer/functions/array/resize_images.py]: Please fix this line

File: chainer/functions/array/scatter_add.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import cupyx
```
Comment (okuta) [inline chainer/functions/array/scatter_add.py]: ditto.

File: chainer/functions/array/spatial_transformer_sampler.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import cupyx
```
Comment (okuta) [inline chainer/functions/array/spatial_transformer_sampler.py]: ditto.

**Comment by kmaehashi** [comment]
Thanks, fixed!

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**